### PR TITLE
Ported to 4.24

### PR DESCRIPTION
**Pull request by aerydna**
Updated to work with 4.24

### CODE REVIEW

File: Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/DRUtils.cpp [hunk 1]
```diff
@@ -708,13 +708,13 @@ namespace DRUtils
             return FoundMeshComp;
         }
 
-        TArray<UActorComponent*> ChildMeshComps = OwnerActor->GetComponentsByClass(UMeshComponent::StaticClass());
-        for (UActorComponent* CheckComp : ChildMeshComps)
+        TArray<UMeshComponent*> ChildMeshComps;
+        OwnerActor->GetComponents<UMeshComponent>(ChildMeshComps);
+        for (auto CheckComp : ChildMeshComps)
```
Comment (thangt) [inline Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/DRUtils.cpp]: You can rename CheckComp to CheckMeshComp here, and keep the code inside the same

File: Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/DRUtils.cpp [hunk 2]
```diff
@@ -748,13 +748,13 @@ namespace DRUtils
             return ValidMeshComps;
         }
 
-        TArray<UActorComponent*> ChildMeshComps = OwnerActor->GetComponentsByClass(UMeshComponent::StaticClass());
-        for (UActorComponent* CheckComp : ChildMeshComps)
+        TArray<UMeshComponent*> ChildMeshComps;
+        OwnerActor->GetComponents<UMeshComponent>(ChildMeshComps);
+        for (auto CheckComp : ChildMeshComps)
```
Comment (thangt) [inline Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/DRUtils.cpp]: same here

File: Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/GroupActorManager.cpp [hunk 3]
```diff
@@ -382,9 +382,9 @@ void AGroupActorManager::UpdateProxyMeshes()
             NewProxyMeshComponent->bIsEditorOnly = true;
             NewProxyMeshComponent->SetCollisionProfileName(UCollisionProfile::NoCollision_ProfileName);
             NewProxyMeshComponent->bHiddenInGame = true;
-            NewProxyMeshComponent->bVisible = !bShouldHideProxyMesh;
+            NewProxyMeshComponent->SetVisibility(!bShouldHideProxyMesh);
             NewProxyMeshComponent->CastShadow = false;
-            NewProxyMeshComponent->PostPhysicsComponentTick.bCanEverTick = false;
+            //NewProxyMeshComponent->PostPhysicsComponentTick.bCanEverTick = false;
```
Comment (thangt) [inline Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/GroupActorManager.cpp:387]: Why do we need to comment out this line?

File: Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/DRUtils.cpp (hunk 1 above)
Comment (aerydna) [reply Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/DRUtils.cpp]: done

File: Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/DRUtils.cpp (hunk 2 above)
Comment (aerydna) [reply Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/DRUtils.cpp]: done

File: Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/GroupActorManager.cpp (hunk 3 above)
Comment (aerydna) [reply Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/GroupActorManager.cpp:387]: PostPhysicsComponentTick is not a member of that class anymore. i can't say the implication of not setting the tick to false. so i prefered to comment it out

**Pull request closed without merging**